go-controller/pkg/network-controller-manager/node_network_controller_manager.go

```diff
@@ -197,6 +201,7 @@ func (ncm *nodeNetworkControllerManager) Start(ctx context.Context) (err error)
 			return fmt.Errorf("failed to start NAD controller: %w", err)
 		}
 	}
+	wg := &sync.WaitGroup{}
```

Comment:
you can just make use of `ncm.wg` instead of new `wg`.

Reply:
yup good catch, this is too old code, needs rebase on latest changes we have done